subprojects/core/src/main/java/org/gradle/caching/internal/BuildCacheServiceProvider.java

```diff
@@ -64,7 +64,7 @@ public BuildCacheService createBuildCacheService() {
         boolean canUseRemoteBuildCache = remote != null && remote.isEnabled();
 
         if (canUseRemoteBuildCache && startParameter.isOffline()) {
-            LOGGER.warn("Remote build cache is disabled when running with --offline.");
+            LOGGER.lifecycle("Remote build cache is disabled when running with --offline.");
```

Comment:
That's what you proposed here: https://github.com/gradle/gradle/pull/1952#commitcomment-21989913